packages/cyberstorm/src/components/LinkLibrary.tsx

```diff
@@ -0,0 +1,63 @@
+import {
```

Comment:
LinkLibrary should be defined by the app that uses these components, since they are the ones who know how the links should be rendered. Half of the point of complicated LinkingProvider is to allow writing components that don't know how the links should be rendered in the end. (The other half is to allow e.g. community link to point to different URLs in different apps.)

Reply:
Alright, I've now done it like so now that the LinkLibrary is defined outside the cyberstorm package. For example now that it's used by the storybook, it's defined in the stories file.